spec/filters/cidr_spec.rb

```diff
@@ -102,5 +104,52 @@
        insist { subject.get("tags") }.nil?
     end
   end
+  
+  describe "Load network list from a file" do
 
+    let(:network_path) {File.join(File.dirname(__FILE__), "..","files","network")}
```

Comment:
Please always put a space after the comma between function call arguments (also line 126 and 142).